lib/fortnox/api/types/document_row.rb

```diff
@@ -22,8 +22,8 @@ class DocumentRow < Fortnox::API::Types::Model
         # DeliveredQuantity Delivered quantity. 14 digits
         attribute :delivered_quantity, Types::Sized::Float[-9_999_999_999_999.9, 9_999_999_999_999.9]
 
-        # Description Description Row description. 50 characters
-        attribute :description, Types::Sized::String[50]
+        # Description Description Row description.
```

Comment:
I guess it is ok to remove it from the comment - but then we should really remove it from all comments just to be consistent 🙂

Reply:
Ye, precicely. We should remove all comments. I can do that in a separate commit :)